src/inputs/dropdown/dropdown.jsx

```diff
@@ -71,6 +71,37 @@ class Dropdown extends React.Component {
         if (!_.isEqual(currentPropsValue, previousPropsValue)) {
             this.setState({ menuIsOpen: false, value: currentPropsValue });
         }
+
+        const {
+            autoScrollSelection,
+            options,
+            selection,
+            selectionMatchProp,
+            value
+        } = this.props;
+        if (selection && autoScrollSelection && this.dropdownMenu && value) {
+            const itemHeight = this.dropdownMenu.getScrollHeight()/_.size(options);
+            const pageSize = this.dropdownMenu.getClientHeight()/itemHeight;
+            const selectionIndex = _.findIndex(options, (o) => {
+                if (selectionMatchProp === 'any') {
+                    const hasValue = _.has(o, 'value');
+                    const hasLabel = _.has(o, 'label');
+
+                    if (!hasValue && !hasLabel) {
+                        return false;
+                    }
+
+                    return (hasValue && o['value'] === value['value'] || o['value'] === value) ||
+                        (hasLabel && o['label'] === value['label'] || o['label'] === value);
+                }
+
+                return o[selectionMatchProp] === value[selectionMatchProp];
+            });
+            const page = Math.floor(selectionIndex/pageSize);
+            if (page >= 0) {
```

Comment:
Nitpick: A blank line in between Lines 99 and 100 and also one between 100 and 101 would be nice but this is absolutely _Not a Blocker_

Reply:
Same here as well.

Reply:
Looks good.  Resolving.